services/app-web/src/pages/StateSubmission/ReviewSubmit/RateDetailsSummarySection.test.tsx

```diff
@@ -704,6 +710,35 @@ describe('RateDetailsSummarySection', () => {
         ).toBeInTheDocument()
     })
 
+    it('renders selected medicaid populations', async () => {
+        renderWithProviders(
+            <RateDetailsSummarySection
+                contract={draftContract}
+                editNavigateTo="rate-details"
+                submissionName="MN-PMAP-0001"
+                statePrograms={statePrograms}
+            />,
+            {
+                apolloProvider: apolloProviderCMSUser,
+                featureFlags: { dsnp: true },
+            }
+        )
+
+        await waitFor(() => {
```

Comment:
@ruizajtruss I see what you're saying with this test passing locally and failing in CI. From the output here, it looks like the test is running its assertions before the page has had a full chance to load. If you add an await for the page heading to be populated that should hopefully fix this

```
const rateName =
            'MCR-MN-0005-SNBC-RATE-20221014-20221014-CERTIFICATION-20221014'

        expect(screen.getByText(rateName)).toBeInTheDocument()
```

The[ test on line 324](https://github.com/Enterprise-CMCS/managed-care-review/blob/b7aff2374678223eb172ed47482a87d186ca8ded/services/app-web/src/pages/StateSubmission/ReviewSubmit/RateDetailsSummarySection.test.tsx#L324) '' is a good example and shows how to set the rate name